src/plugins/intel_gpu/thirdparty/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ if(ENABLE_ONEDNN_FOR_GPU)
         set(ONEDNN_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/onednn_gpu_build/")
         set(ONEDNN_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/onednn_gpu_install/")
         set(ONEDNN_PREFIX_DIR "${CMAKE_CURRENT_BINARY_DIR}/onednn_gpu_root")
-        execute_process(COMMAND git apply --verbose ../onednn_gpu.patch
+        execute_process(COMMAND git apply ../onednn_gpu.patch OUTPUT_QUIET ERROR_QUIET
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#issuecomment-1313402310
That error message occurs when trying to patch code that has already been patched again. I updated not to be displayed to prevent confusion.